src/java/edu/ucsd/library/dams/triple/TripleStoreUtil.java

```diff
@@ -320,6 +335,49 @@ private static void loadRDF( InputStream in, boolean deleteFirst,
 		{
 			throw new TripleStoreException( "Error processing triples", ex );
 		}
+
+		return null;
+	}
+	public static Set<String> validateModel( Model model, Set<String> validClasses,
```

Comment:
If our list of validations continues to grow beyond core class/predicate matching, I wonder if this belongs in its own class, with each of the validation checks being in a separate method. I know we'll eventually port this to Ruby, so I'm not strongly opinionated on this.